src/controllers/index.js

```diff
@@ -3,9 +3,19 @@ const router = express.Router();
 
 const home = require('./home');
 const signUp = require('./sign_up');
+const signIn = require('./sign_in');
+const businessPage = require('./business_page');
+const userProfile = require('./user_profile');
+const error = require('./error');
 
+// router Home Page
```

Comment:
these comments should be in the same line as the code and not above it. It makes the code a lot easier to read.